bitcoin/src/psbt/mod.rs

```diff
@@ -336,16 +335,15 @@ impl Psbt {
                 Ok((Message::from(sighash), hash_ty))
             }
             Wpkh => {
-                let script_code = ScriptBuf::p2wpkh_script_code(spk).ok_or(SignError::NotWpkh)?;
+                let script_code = spk.p2wpkh_script_code().ok_or(SignError::NotWpkh)?;
                 let sighash =
                     cache.segwit_signature_hash(input_index, &script_code, utxo.value, hash_ty)?;
                 Ok((Message::from(sighash), hash_ty))
             }
             ShWpkh => {
-                let script_code = ScriptBuf::p2wpkh_script_code(
-                    input.redeem_script.as_ref().expect("checked above"),
-                )
-                .ok_or(SignError::NotWpkh)?;
+                let script_code = input.redeem_script.as_ref().expect("checked above")
+                    .p2wpkh_script_code()
+                    .ok_or(SignError::NotWpkh)?;
```

Comment:
I'm definitely glad we run `rustfmt` in CI, you're a menace :)

Reply:
Why? This piece of code does one thing: convert a redeem script to a script code. When reading the code, that's a single step. Ideally it would be one line of code.

In the code I wrote, 

- the first line gets the `redeem_script` field (the `.as_ref().expect("checked above")` doesn't contain any information and how I wrote it, your mind can just ignore that part while you skim the code),
- the second line calls the `p2wpkh_script_code` method on it.

So that's the bulk of the logical steps. And then because we are good citizens, we add some explicit error handling, sure that's fine.

These are the two/three pieces of information that matter.

To write this as
```rust
                let script_code = input
                    .redeem_script
                    .as_ref()
                    .expect("checked above")
                    .p2wpkh_script_code()
                    .ok_or(SignError::NotWpkh)?;
```

means you have so much vertical space for only so little information. The first, third and fourth line are just wasted vertical space.

:shrug: 

Vertical space matters, the human mind is a lot better understanding things when you can look at them in the same view. Scrolling breaks that. I personally have a lot of vertical space, 75 lines, but I have had colleagues that had only 40. You can lack vertical space for various reasons like larger font because of bad eyesight, temporary work in a VPSs with tmux, when reviewing code on a phone, ...

</rant><anticipation-for-next-occasion>

Reply:
I have no problem with your layout except that it is different. Same as is often the case, like many things in formatting, I don't care what it is as long as it is uniform. My brain works better when things are uniform, I use less cycles to achieve the same thing and I catch mistakes easier. That's just my personal experience from watching my brain function (and malfunction).